web-local/README.md

```diff
@@ -0,0 +1,17 @@
+# web-local
+
+This folder contains the local frontend implemented with TypeScript and [SvelteKit](https://kit.svelte.dev). 
+
+## Running in development
+
+To run the web server with an embedded runtime:
+
+1. In `web-local/build-tools/postinstall_runtime.sh`, edit the `RUNTIME_VERSION` constant with the commit hash of the runtime version you want to use.
```

Comment:
Probably good to add the instructions to run the job to pull latest changes to go.

Reply:
I think that belongs in `runtime/README.md` instead